change/@fluentui-react-motion-components-preview-b271a894-6660-439a-aef6-8f008b144e56.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33069&commitId=a355b3e009460e61514ddce8901d0c44b46deba1&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|14| Changed|
|Avatar Converged.badgeMask.chromium.png|18| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay High Contrast.chromium.png|2251| Changed|
|Drawer.Full Overlay Dark Mode.chromium.png|2798| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>